### PR TITLE
Test.

### PR DESCRIPTION
**Pull request by jlahoda**

### CODE REVIEW

File: src/java.base/share/classes/sun/launcher/LauncherHelper.java [hunk 1]
```diff
@@ -152,7 +152,7 @@ private LauncherHelper() {}
     @SuppressWarnings("fallthrough")
     static void showSettings(boolean printToStderr, String optionFlag,
             long initialHeapSize, long maxHeapSize, long stackSize) {
-
```
Comment (jlahoda) [inline src/java.base/share/classes/sun/launcher/LauncherHelper.java:155]: Jackpot:
warning: Thread Dump Stack

Comment (jlahoda) [inline src/java.base/share/classes/sun/launcher/LauncherHelper.java:155]: Jackpot:
warning: Thread Dump Stack

Comment (jlahoda) [inline src/java.base/share/classes/sun/launcher/LauncherHelper.java:155]: Jackpot:
warning: Thread Dump Stack

File: src/jdk.compiler/share/classes/com/sun/tools/javac/api/JavacTaskImpl.java [hunk 2]
```diff
@@ -127,6 +127,7 @@ public void addModules(Iterable<String> moduleNames) {
 
     @Override @DefinedBy(Api.COMPILER)
     public void setProcessors(Iterable<? extends Processor> processors) {
+        Thread.dumpStack();
```
Comment (jlahoda) [inline src/jdk.compiler/share/classes/com/sun/tools/javac/api/JavacTaskImpl.java:130]: Jackpot:
warning: Thread Dump Stack

Comment (jlahoda) [inline src/jdk.compiler/share/classes/com/sun/tools/javac/api/JavacTaskImpl.java:130]: Jackpot:
warning: Thread Dump Stack

File: src/java.base/share/classes/sun/launcher/LauncherHelper.java (hunk 1 above)
Comment (jlahoda) [inline src/java.base/share/classes/sun/launcher/LauncherHelper.java:155]: Jackpot:
warning: Thread Dump Stack

Comment (jlahoda) [inline src/java.base/share/classes/sun/launcher/LauncherHelper.java:155]: Jackpot:
warning: Thread Dump Stack

File: src/jdk.compiler/share/classes/com/sun/tools/javac/api/JavacTaskImpl.java (hunk 2 above)
Comment (jlahoda) [inline src/jdk.compiler/share/classes/com/sun/tools/javac/api/JavacTaskImpl.java:130]: Jackpot:
warning: Thread Dump Stack

File: src/java.base/share/classes/sun/launcher/LauncherHelper.java (hunk 1 above)
Comment (jlahoda) [inline src/java.base/share/classes/sun/launcher/LauncherHelper.java:155]: Jackpot:
warning: Thread Dump Stack

Comment (jlahoda-jackpot) [inline src/java.base/share/classes/sun/launcher/LauncherHelper.java:155]: Jackpot:
warning: Thread Dump Stack

File: test/langtools/tools/javac/modules/AnnotationsOnModules.java [hunk 3]
```diff
@@ -22,8 +22,8 @@
  */
 
 /*
- * @test
  * @bug 8159602 8170549 8171255 8171322
```
Comment (jlahoda-jackpot) [inline test/langtools/tools/javac/modules/AnnotationsOnModules.java:25]: Jackpot:
warning: Incorrect tag order

File: test/jdk/java/util/regex/RegExTest.java [hunk 4]
```diff
@@ -22,8 +22,8 @@
  */
 
 /**
- * @test
  * @summary tests RegExp framework (use -Dseed=X to set PRNG seed)
```
Comment (jlahoda-jackpot) [inline test/jdk/java/util/regex/RegExTest.java:25]: Jackpot:
warning: Incorrect tag order

File: test/langtools/tools/javac/modules/AnnotationsOnModules.java (hunk 3 above)
Comment (jlahoda-jackpot) [inline test/langtools/tools/javac/modules/AnnotationsOnModules.java:25]: Jackpot:
warning: Incorrect tag order

File: test/jdk/java/util/regex/RegExTest.java (hunk 4 above)
Comment (jlahoda-jackpot) [inline test/jdk/java/util/regex/RegExTest.java:25]: Jackpot:
warning: Incorrect tag order

File: src/java.base/share/classes/sun/launcher/LauncherHelper.java (hunk 1 above)
Comment (jlahoda-jackpot) [inline src/java.base/share/classes/sun/launcher/LauncherHelper.java:155]: Jackpot:
warning: Thread Dump Stack

File: test/jdk/java/util/regex/RegExTest.java (hunk 4 above)
Comment (jlahoda-jackpot) [inline test/jdk/java/util/regex/RegExTest.java:25]: Jackpot:
warning: Incorrect tag order

File: test/langtools/tools/javac/modules/AnnotationsOnModules.java (hunk 3 above)
Comment (jlahoda-jackpot) [inline test/langtools/tools/javac/modules/AnnotationsOnModules.java:25]: Jackpot:
warning: Incorrect tag order

File: src/java.base/share/classes/sun/launcher/LauncherHelper.java (hunk 1 above)
Comment (jlahoda-jackpot) [inline src/java.base/share/classes/sun/launcher/LauncherHelper.java:155]: Jackpot:
warning: Thread Dump Stack

**Pull request closed without merging**